db/schema.rb

```diff
@@ -10,34 +10,33 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2021_08_19_160319) do
+ActiveRecord::Schema.define(version: 2022_03_15_104737) do
 
-  create_table "annotations", id: :integer, force: :cascade do |t|
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
+  create_table "annotations", id: :serial, force: :cascade do |t|
     t.integer "question_id"
     t.integer "org_id"
     t.text "text"
     t.integer "type", default: 0, null: false
     t.datetime "created_at"
     t.datetime "updated_at"
     t.string "versionable_id", limit: 36
-    t.index ["org_id"], name: "fk_rails_aca7521f72"
     t.index ["question_id"], name: "index_annotations_on_question_id"
     t.index ["versionable_id"], name: "index_annotations_on_versionable_id"
   end
 
-  create_table "answers", id: :integer, force: :cascade do |t|
+  create_table "answers", id: :serial, force: :cascade do |t|
```

Comment:
Looks like some extra schema changes leaked into this PR. Just add the new `helpdesk_email` column to the Orgs table

Reply:
That already exists 

create_table "orgs", id: :serial, force: :cascade do |t|
    t.string "name"
    t.string "abbreviation"
    t.string "target_url"
    t.datetime "created_at", null: false
    t.datetime "updated_at", null: false
    t.boolean "is_other", default: false, null: false
    t.integer "region_id"
    t.integer "language_id"
    t.string "logo_uid"
    t.string "logo_name"
    t.string "contact_email"
    t.integer "org_type", default: 0, null: false
    t.text "links"
    t.string "contact_name"
    t.boolean "feedback_enabled", default: false
    t.text "feedback_msg"
    t.boolean "managed", default: false, null: false
    **t.string "helpdesk_email"**
  end

Reply:
Reverted  create_table "answers", id: :serial, force: :cascade do |t| to 
 create_table "answers", id: :integer, force: :cascade do |t|.